counterpoll/main.py

```diff
@@ -382,6 +388,19 @@ def disable(ctx):
     fc_info['FLEX_COUNTER_STATUS'] = 'disable'
     ctx.obj.mod_entry("FLEX_COUNTER_TABLE", "FLOW_CNT_ROUTE", fc_info)
 
+
+@cli.command()
+@click.pass_context
+def delay(ctx):
```

Comment:
A reset_delay knob can be helpful addition along with the delay knob that is being added here.

You can use that reset_delay knob in the failure handling path of fast-reboot when it fails w/ `EXIT_COUNTERPOLL_DELAY_FAILURE`

Reply:
@vaibhavhd It is a reasonable request. However, it is out of scope of bug fix PR and I do not change the existing desing. I suggest we open an issue to handle cleanup path for this scenario.

Reply:
Ok, a different PR is acceptable to me too. Can you open a new issue and can you please work on this cleanup path PR as well?

Reply:
@vaibhavhd Issue is created https://github.com/sonic-net/sonic-buildimage/issues/20270